### PR TITLE
[GUIDE] Deposits & bonds explained

### PR DESCRIPTION
**Pull request by havan**
New user guide page that explains deposits and bonds in Camino Network.

Deployed [here](https://playground.docs.camino.network/guides/deposits-and-bonds/index.html).

Needs #71 to be merged first.

### CODE REVIEW

File: docs/faq/validator.md [hunk 1]
```diff
@@ -5,47 +5,56 @@ description: Frequently asked questions about validation on the camino and colum
...
-  The last condition is omitted for Validators who secure the network from the start. To secure the Columbus Testnet, one must only be a market participant in the travel industry.
+- Holds a minimum of 100,000 Camino (CAM) tokens in their wallet.
+- Has undergone and passed a Know-Your-Customer (KYC) and Know-Your-Business (KYB) verification process.
+- Has been elected as a validator by the validator consortium through a voting process.
 
-## Staking Parameters on Camino
+:::note
 
-When a validator is done validating the Camino Mainnet, it receives back the Camino tokens it staked. It may receive a reward for helping to secure the network. A validator only receives a validation reward if it is sufficiently responsive and correct during the validation period.
+Please note, the last requirement is not applicable for validators who are securing the network from its inception. For securing the Columbus Testnet, it's sufficient to be a market participant in the travel industry.
```
Comment (evlekht) [inline docs/faq/validator.md:18]: How's that? I don't think that we'll have power to do something like add/remove c-members without voting, so any validator can be removed later. Isn't this right? May be I got this line wrong

File: docs/guides/deposits-and-bonds.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+This bonding mechanism is used to ensure that the person proposing the proposal is invested in its success and has a stake in the outcome. It also helps prevent spamming or low-quality proposals from being submitted to the DAO, as users have to put up a bond to propose a proposal. Overall, the use of bonds in Camino Network helps to ensure the integrity and security of the network.
+
+## Conclusion
+
+Overall, the deposit and bond features in Camino Network provide a simple and secure way for users to earn rewards and participate in the decision-making process of the network. The fixed staking amount and the requirement for validators to be verified create a fair and democratic environment for all members.
+
+:::info DEPOSITS & BONDS ARE LOCKED TOKENS
+
+It's important to note that tokens that are bonded, whether for validator staking or DAO proposals, cannot be used for paying transaction fees. This is because the tokens are locked and cannot be moved or transferred during the bond period.
```
Comment (evlekht) [inline docs/guides/deposits-and-bonds.md:59]: They also can't be exported

File: docs/faq/validator.md (hunk 1 above)
Comment (havan) [reply docs/faq/validator.md:18]: Maybe I need better wording. What I was trying to say there was that the initial c-members (in genesis block) would not go through a  *voting process* and they are exempt from that *last requirement*.

File: docs/guides/deposits-and-bonds.md (hunk 2 above)
Comment (havan) [reply docs/guides/deposits-and-bonds.md]: Updated to mention that also.

File: docs/faq/validator.md (hunk 1 above)
Comment (havan) [reply docs/faq/validator.md]: Updated this. Please note that this commit is part of the PR #71. I don't know what will happen to these comments when #71 is merged and I rebase this PR to the new dev. :)

**Pull request merged**